plugins/gs.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gs
+spec:
+  version: v0.5.0
+  homepage: https://github.com/giantswarm/kubectl-gs
+  shortDescription: Giant Swarm plug-in for kubectl
+  description: |
+    Simplifies creating clusters and node pools via Giant Swarm Kubernetes
+    control planes, as well as installing app catalogs and apps.
+  platforms:
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/giantswarm/kubectl-gs/releases/download/v0.5.0/kubectl-gs_0.5.0_darwin_amd64.tar.gz
```

Comment:
inactionable hre : your --help message shows 
 `kubectl-gs` which won't work once it's installed as a plugin. you might wanna correct that in an upcoming version

Reply:
Thanks for the hint! Noted for the next release.